packages/js/api/src/framework/__tests__/payload-transformer.spec.ts

```diff
@@ -0,0 +1,383 @@
+import {Order, BillingOrderAddress, ShippingOrderAddress, OrderLineItem, OrderShippingLine, OrderFeeLine, OrderTaxRate, OrderCouponLine, MetaData, OrderRefundLine} from "../../models";
+import {createBillingAddressTransformer, createOrderTransformer, createShippingAddressTransformer} from "../../repositories/rest/orders/transformer";
+
+const jsonPayloadFull = JSON.stringify({
```

Comment:
Since this is specifically testing the order model's REST API transformation, it might make more sense to live in `src/repositories/rest/orders/__tests__`.